### ISSUE TITLE
sent2vec in Gensim

### ISSUE THREAD

**Issue by bhargavvader** [RaRe-Technologies/gensim#1376]
The EPFL ML group have come up with a sentence embedding technique, [sent2vec](https://arxiv.org/abs/1703.02507). The repo with the code can be found [here](https://github.com/epfml/sent2vec).

It would be interesting to have this feature in Gensim, either with a wrapper or otherwise.

**Comment by bhargavvader** [comment]
@menshikh-iv could you please add the appropriate labels for this?

**Comment by menshikh-iv** [comment]
@bhargavvader of course :)

**Comment by souravsingh** [comment]
I am interested in working on the issue.

**Comment by colmantse** [comment]
I am also interested, but I am new on GitHub. Would be great if this could be my first contribution.

**Comment by martinjaggi** [comment]
happy to help if any questions should come up. note that our current code is very close to fasttext (though unsupervised), so integration should be not too hard

**Comment by simonhessner** [comment]
Just wanted to ask for updates regarding sent2vec in gensim. Is somebody currently working on it?

**Comment by menshikh-iv** [comment]
@simonhessner this is almost done in #1619, will be merged soon (I hope).

**Comment by SriniNaga** [comment]
@menshikh-iv  Is  this feature(Sent2Vec) available  in Gensim now?

**Comment by menshikh-iv** [comment]
@SriniNaga sorry, still not merged, will be available in `3.7.0`

**Comment by wandabwa2004** [comment]
@menshikh-iv  any timelines for version 3.7.0?

**Comment by prerna135** [comment]
@menshikh-iv @piskvorky Sent you an email regarding this.

**Comment by cstayyab** [comment]
@menshikh-iv @piskvorky any updates on this issue and the merge conflict?

**Comment by piskvorky** [comment]
@cstayyab It's up to @prerna135 .

**Comment by tmthyjames** [comment]
any update on this? love the blog post on it! 

gensim is awesome! keep up the great work.

**Comment by MarcosGrzeca** [comment]
News?

Thank you :)

**Comment by 0x01h** [comment]
Any improvements on this issue?

**Comment by mpenkov** [comment]
No. The original author has abandoned the PR, and I don't have the bandwidth to take it on myself right now. If anyone is able and willing to push it over the line, please go ahead by all means.
